ChangeLog

```diff
@@ -343,6 +343,10 @@ Release date: 2017-04-13
 
       Fixes #59
 
+    * Ignore unknown members when one of the inferred owners is ignored for
```

Comment:
I will move this to the latest part of the Changelog once this PR gets some attention: no need to chase after the versions before that.